hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -88,8 +88,9 @@ public void testDemo() throws Exception {
   }
 
   private void setupDemo() throws Exception {
-    List<String> cmds = new ImmutableList.Builder<String>().add("hdfs dfsadmin -safemode wait") // handle NN going into
-                                                                                                // safe mode at times
+    List<String> cmds = new ImmutableList.Builder<String>()
+        .add("hdfs dfsadmin -safemode wait") // handle NN going into safe mode at times
```

Comment:
Could we keep hdfs data within the container itself? instead of formatting.. This adds more steps to the demo when trying by hand also.. more chances of encountering some issue with hdfs itself.